how-it-works/2_chainkeytech/04_chain_key_tokens.subpage.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Chain-key Tokens
```

Comment:
"Chain-Key Tokens" or "Chain-key tokens"?

Reply:
We decided for "chain-key tokens" when not in a heading. In the heading, it is a matter of style (Chicago or not) whether to caps the "Key".
"Chain-Key Bitcoin" is always all caps first letters as it is a proper noun (as token name).